website/docs/d/service_level_alert_helper.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_service_level_alert_helper"
+sidebar_current: "docs-newrelic-datasource-service-level-alert-helper"
+description: |-
+  Helper to set up alerts on Service Levels.
+---
+
+# Data Source: newrelic\_service\_level\_alert\_helper
+
+Use this data source to obtain the necessary fields to set up alerts on your service levels. It can be used for a `custom` alert_type in order to set up an alert with custom tolerated budget consumption and custom evaluation period or for recommended ones like `fast_burn`.
+
+## Example Usage
+
+Firstly set up your service level objective, we recommend to use local variables for the `target` and `time_window.rolling.count`, as they are also necessary for the helper.
+
+```hcl
+locals {
+  foo_target = 99.9
+  foo_period = 28
+}
+
+resource "newrelic_service_level" "foo" {
+    guid = "MXxBUE18QVBQTElDQVRJT058MQ"
+    name = "Latency"
+    description = "Proportion of requests that are served faster than a threshold."
+
+    events {
+        account_id = 12345678
+        valid_events {
+            from = "Transaction"
+            where = "appName = 'Example application' AND (transactionType='Web')"
+        }
+        good_events {
+            from = "Transaction"
+            where = "appName = 'Example application' AND (transactionType= 'Web') AND duration < 0.1"
+        }
+    }
+
+    objective {
+        target = local.foo_target
+        time_window {
+            rolling {
+                count = local.foo_period
+                unit = "DAY"
+            }
+        }
+    }
+}
+```
+Then use the helper to obtain the necessary fields to set up an alert on that Service Level.
+
+```hcl
+
+data "newrelic_service_level_alert_helper" "foo_custom" {
+    alert_type = "custom"
+    sli_guid = "MXxBUE18QVBQTElDQVRJT058MQ"
```

Comment:
`MXxBUE18QVBQTElDQVRJT058MQ` is the GUID of the service/application over which we want to track the SLI, as in (see above)

```
resource "newrelic_service_level" "foo" {
    guid = "MXxBUE18QVBQTElDQVRJT058MQ"
    name = "Latency"
    ...
```
and, if I'm understanding this right, the value of `sli_guid` should be another value.

So, in order to avoid inducing the reader to error,  I'd either use explicitly another GUID value , or even maybe use `newrelic_service_level.sli_guid` if it makes sense for the example purpose?